rules/cloud/azure/azure_aad_secops_ca_policy_updatedby_bad_actor.yml

```diff
@@ -0,0 +1,24 @@
+title: CA Policy Updated by Non Approved Actor
+id: 50a3c7aa-ec29-44a4-92c1-fce229eef6fc
+description: Monitor and alert on conditional access changes. Is Initated by (actor) approved to make changes? Review Modified Properties and compare "old" vs "new" value.
+author: Corissa Koopmans, '@corissalea'
+date: 2022/07/19
+references:
+  - https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/security-operations-infrastructure#conditional-access
+logsource:
+  product: azure
+  service: auditlogs
+detection:
+  keywords:
+    - Update conditional access policy
+  condition: selection
```

Comment:
"keywords" must be used in the condition. Change the "condition" part to be 

`condition: keywords`

Reply:
fixed it for her